src/rp2_common/pico_cyw43_driver/CMakeLists.txt

```diff
@@ -56,6 +56,34 @@ if (EXISTS ${PICO_CYW43_DRIVER_PATH}/${CYW43_DRIVER_TEST_FILE})
             hardware_exception
             )
 
+    if (CYW43_DEFAULT_PIN_WL_REG_ON)
```

Comment:
Need to add PICO_CMAKE_CONFIG stuff for these

Reply:
do you want to do that as part of this?

Reply:
Also, why do we want this settable from CMake vs board header?